Yearn-agents/InstantWithdrawal/package.json

```diff
@@ -20,17 +24,16 @@
   },
   "dependencies": {
     "axios": "^0.21.4",
-    "forta-agent": "^0.0.39",
-    "ganache-core": "^2.13.2"
+    "forta-agent": "^0.1.9",
+    "ganache": "^7.4.3"
   },
   "devDependencies": {
```

Comment:
It appears you have run `prettier`, but I don't see it installed under `devDependencies`.

Reply:
that's right. because I have it installed globally. so I forgot to add it here. added 👍